workflow/rules/inference.smk

```diff
@@ -185,14 +192,18 @@ rule inference_interpolator:
         reftime_to_iso=lambda wc: datetime.strptime(
             wc.init_time, "%Y%m%d%H%M"
         ).strftime("%Y-%m-%dT%H:%M"),
-        forecaster_run_id=lambda wc: _get_forecaster_run_id(wc.run_id),
+        forecaster_run_id=lambda wc: (
+            "null"
+            if RUN_CONFIGS[wc.run_id].get("forecaster") is None
+            else RUN_CONFIGS[wc.run_id]["forecaster"]["run_id"]
```

Comment:
```suggestion
            else RUN_CONFIGS[wc.run_id.split("-")[1]]
```

Reply:
I don't think this works. But maybe the problem is indeed that we haven't generated the run_id for the forecaster in the interpolator config (mind you, as long as you don't pop the forecaster, the whole forecaster config is still present in RUN_CONFIGS, which is what i used here).